### PR TITLE
pass allow_ssl_requests_only to storage module

### PR DESCRIPTION
**Pull request by mcalhoun**
## what

* Create a flag to require the underlying s3 bucket to only support access via tls

## why

* AWS Foundational Security Best Practices

### CODE REVIEW

File: variables.tf [hunk 1]
```diff
@@ -130,3 +130,9 @@ variable "create_access_log_bucket" {
   default     = false
   description = "A flag to indicate if a bucket for s3 access logs should be created"
 }
+
+variable "allow_ssl_requests_only" {
```
Comment (osterman) [inline variables.tf:134]: What about simply "require_ssl" ?

Comment (mcalhoun) [reply variables.tf:134]: This is the name already used in the underlying s3 bucket module (not my doing) so I kept it consistent

**Review by nitrocode — APPROVED** [review]

**Pull request merged**